notifier/settings.py

```diff
@@ -193,6 +193,9 @@
 # before it's replaced with a new one.
 CELERYD_MAX_TASKS_PER_CHILD = int(os.getenv('CELERYD_MAX_TASKS_PER_CHILD', 100))
 
+DEFAULT_PRIORITY_QUEUE = 'notifier.default.queue'
```

Comment:
I think this can just be notifier

Reply:
Can you read this one from the environment:
https://github.com/edx/configuration/pull/4964/files

Since you are setting it here for the workers, we'll want to make sure both get updated if someone changes it, so I dont think we can hardcode this one.

Reply:
@syed-awais-ali  it looks like this still needs to be updated per Cory's suggestions.

Reply:
@feanil, sorry. I thought this was pushed. It was until now that I saw that why isn't github reflecting the changes i made. Then i realized that i had committed the code but somehow I missed pushing it upstream.